### ISSUE TITLE
Cannot get logs for processes that are stopped

### ISSUE THREAD

**Issue by sarm333** [spauny/roboscript#1]
Hey,

Very useful tool! It works for my current project but there's only one minor niggle. When you stop a already running process, you are no longer to grab the logs any more via the endpoint as the process ID will not exist anymore. Is there nay way to store the logs somewhere after its been killed?

**Comment by sarm333** [comment]
It would be great if the process IDs still exist after it stops for you to still view the logs after. Maybe these can then be deleted once you stop the tomcat server. Another useful endpoint would be to have one that lists the processes that are currently running. That way, you can just copy and paste the Id's into the log endpoint to see whats going on with that process.

**Comment by spauny** [comment]
Yeah, basically now when you stop the process, the item is deleted from the underlying map holder... we just have to remove the removal :D ... ok, waiting your pull request :D
